src/js/modules/msg.js

```diff
@@ -200,26 +200,26 @@
 
 
 // constant for "same tab as me"
-var SAME_TAB = -1000;  // was -Infinity, but JSON.stringify() + JSON.parse() don't like that value
+const SAME_TAB = -1000;  // was -Infinity, but JSON.stringify() + JSON.parse() don't like that value
 
 // run-time API:
 // variable + exported function to change it, so it can be mocked in unit tests
 /* global chrome */
-var runtime = ('object' === typeof(chrome)) && chrome.runtime;
+const runtime = (typeof chrome === 'object') && chrome.runtime;
 // the same for devtools API:
-var devtools = ('object' === typeof(chrome)) && chrome.devtools;
+const devtools = (typeof chrome === 'object') && chrome.devtools;
 
 // utility function for looping through object's own keys
 // callback: function(key, value, obj) ... doesn't need to use all 3 parameters
 // returns object with same keys as the callback was invoked on, values are the
 //   callback returned values ... can be of course ignored by the caller, too
 function forOwnProps(obj, callback) {
-  if ('function' !== typeof(callback)) {
-    return;
+  if (typeof callback !== 'function') {
+    return null;
   }
-  var res = {};
-  for (var key in obj) {
-    if (obj.hasOwnProperty(key)) {
+  const res = {};
+  for (const key in obj) { // eslint-disable-line no-restricted-syntax
```

Comment:
hmm, now I know why I did this - eslint says prefer-const - key is never reassigned. Use 'const' instead.
I guess it could be interpreted as a new constant being initialized in each iteration of for.
http://stackoverflow.com/questions/31987465/ecmascript-2015-const-in-for-loops